src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Frameworks/ITargetFramework.cs

```diff
@@ -3,9 +3,9 @@
 using System;
 using System.Runtime.Versioning;
 
-namespace Microsoft.VisualStudio.ProjectSystem.VS.Tree.Dependencies.CrossTarget
+namespace Microsoft.VisualStudio.ProjectSystem
 {
-    internal interface ITargetFramework : IEquatable<ITargetFramework>,
+    public interface ITargetFramework : IEquatable<ITargetFramework>,
```

Comment:
Do we have to make this type public? Can we use IVT instead?

Reply:
see the comments below.  what is IVT?

Reply:
IVT = InternalsVisibleTo attribute

Reply:
I don't think it makes sense, you have to white list all consumers here.